src/SFA.DAS.LearnerDataMismatches.Domain/AcademicYear.cs

```diff
@@ -4,28 +4,50 @@ namespace SFA.DAS.LearnerDataMismatches.Domain
 {
     public struct AcademicYear
     {
-        public DateTime StartingDate { get; }
-        public int ShortRepresentation { get; }
-
-        public override string ToString() =>
-            $"{StartingDate.Year} - {StartingDate.Year + 1}";
-
-        public AcademicYear PreviousAcademicYear => this - 1;
         public AcademicYear(DateTime instant)
         {
-            var year = instant.Month >= 8 ? instant.Year : instant.Year - 1;
+            var year = instant.Month >= FirstMonthOfAcademicYear ? instant.Year : instant.Year - 1;
```

Comment:
Since you have created a constant, perhaps you can use that in the StartingDate assignment as well.

Reply:
D'oh, yep good idea.